Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# Reaction Forces Templates

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added Reaction Forces templates from [Community PR 3185 (Tiny-DM)](https://github.com/official-antistasi-community/A3-Antistasi/pull/3185)

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:
Load RF CDLC;
Load one of the compatible RF templates (Most of the vanilla ones);
Enable RF CDLC in additional content;
Ensure RF assets are being used

********************************************************
Notes:
